### DISCUSSION TITLE
Wechaty matrix bridge in China

### DISCUSSION THREAD

**Discussion by xkevinx** [wechaty/wechaty#2220] [Q&A]
Hello!

I would like to add your Wechat bridge to my Matrix server, and I live in China. As my server is abroad, would it be seen by WeChat? I wouldn 't like to have my account blocked or Wechat Pay deactivated because of that 😅

Thanks!

**Comment by huan** [comment]
I believe that the answer to your question depends on which [Wechaty Puppet](https://wechaty.js.org/docs/puppet-providers/) you are using.

CC @wechaty/matrix

**Reply by xkevinx** [reply]
thanks!

**Comment by lprintf** [comment]
> Hello!
> 
> I would like to add your Wechat bridge to my Matrix server, and I live in China. As my server is abroad, would it be seen by WeChat? I wouldn 't like to have my account blocked or Wechat Pay deactivated because of that 
> 
> Thanks!

You can use Wechat bridge in any computer which can access internet, even in personal computer without public IP. (It's not sure, if the computer can't access the China domestic public network.) The risk of account blocked exists, and it is irregular.  Using wecom token can reduse the risk.

**Reply by xkevinx** [reply]
I see, thanks!
